commands/handbook.py

```diff
@@ -0,0 +1,71 @@
+from commands.base import Command
+from discord import Embed
+from helpers import CommandFailure
+
+import bs4, re, requests
+
+
+def subject_details(code):
+    code = code.upper()
+
+    page = requests.get('https://www.handbook.unsw.edu.au/undergraduate/courses/2019/' + code)
+
+    if page.status_code != 200:
+        page = requests.get('https://www.handbook.unsw.edu.au/postgraduate/courses/2019/' + code)
+
+        if page.status_code != 200:
+            return None
+
+    soup = bs4.BeautifulSoup(page.text, features='lxml')
+
+    course_title = soup.find('span', attrs={'data-hbui' : 'module-title'}).string.strip()
+    course_offerings = soup.find('strong', string=re.compile('Offering Terms')).parent.contents[3].string.strip()
+
+    course_conditions_tag = soup.find('div', id='readMoreSubjectConditions')
+
+    course_conditions = 'None'
+
+    if course_conditions_tag is not None:
+        for s in course_conditions_tag.contents[1].contents[1].strings:
+            course_conditions += s
+
+    course_conditions = course_conditions.strip()
+
+    course_description_tag = soup.find('div', id='readMoreIntro')
+
+    course_description = ''
+
+    if len(course_description_tag.contents[1].contents) > 1 and course_description_tag.contents[1].contents[1].name == 'p':
+        for s in course_description_tag.contents[1].contents[1].strings:
+            course_description += s
+    else:
+        course_description = course_description_tag.contents[1].contents[0].string
+
+    course_description = course_description.strip()
+
+    return {
+        'title' : course_title,
+        'description' : course_description,
+        'offerings' : course_offerings,
+        'conditions' : course_conditions,
+        'link' : 'https://www.handbook.unsw.edu.au/undergraduate/courses/2019/' + code
+    }
+
+
+class Handbook(Command):
+    desc = "This command scrapes entries in the UNSW handbook"
+
+    def eval(self, arg):
```

Comment:
Please make the argument variable more descriptive as the help message generated uses this in the help message. Something like `course_code` should suffice.